app/models/user.rb

```diff
@@ -1300,7 +1301,7 @@ def delegate_role
       is_active: true,
       group: self.region,
       user: self,
-      is_lead: delegate_status == RolesMetadataDelegateRegions.statuses[:senior_delegate],
+      is_lead: false,
```

Comment:
I don't think so, because this should be false for all delegate roles who aren't senior/regional delegate. Even though we can do something like `!delegate_status_migrated`, if we migrate trainee delegate alone in the next phase (which we won't anyway), `is_lead` will become true for trainee delegates which is not correct. Currently since we already migrated all the lead delegates, I'm hard-coding the boolean.

Reply:
Okay, thanks for the explanation :)